src/main/java/com/monumental/util/csvparsing/CsvMonumentConverterResult.java

```diff
@@ -77,4 +86,220 @@ public void addTag(Tag tag) {
 
         this.tags.add(tag);
     }
+
+    /**
+     * Validates this CsvMonumentConverterResult
+     * If any of the validations fail, this CsvMonumentConverterResult is considered invalid
+     * @return ValidationResult - ValidationResult object representing the result of the validation
+     */
+    public ValidationResult validate() {
```

Comment:
I decided to move the validation out of `Monument` and place it here instead. It makes much more sense to me to do it this way before we ever create a `Monument`.

Reply:
Plus, this is a friendly way to get the validation results back to the front-end and make them easy to use.